sklearnex/tests/test_patching.py

```diff
@@ -20,6 +20,7 @@
 import re
 import subprocess
 import sys
+import warnings
```

Comment:
is it used?

Reply:
good eye, there was something before that used it, but available_if ruined it.